README.md

```diff
@@ -7,6 +7,7 @@ It resembles well known tools such as _supervisord_, _runit_, or _s6_, in that i
 easily manage non-system processes independently from the system services, but it was
 designed with unique features that help with more specific use cases.
 
+  - [Installation](#installation)
   - [General model](#general-model)
   - [Layer configuration examples](#layer-configuration-examples)
   - [Using pebble](#using-pebble)
```

Comment:
Also, as a drive-by, let's change this "Using Pebble" (that is, "Pebble" capitalized).

Reply:
Fixed this and other occurrences of this typo.